.github/workflows/static-checks.yml

```diff
@@ -115,3 +116,33 @@ jobs:
     - uses: actions/checkout@v4
     - name: REUSE Compliance Check
       uses: fsfe/reuse-action@v5
+
+  lint-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Install Commit-Lint
+        run: npm install --save-dev @commitlint/cli @commitlint/config-conventional
```

Comment:
Better to use actions from marketplace than installing tools manually...

See https://github.com/marketplace/actions/semantic-pull-request for example.

Reply:
Sure, Will take a look into this :)